SUASSystem/SUASSystem/image_processing.py

```diff
@@ -48,6 +48,7 @@ def run_img_proc_process(logger_queue, location_log, targets_to_submit, interop_
 def run_autonomous_img_proc_process(logger_queue, location_log, interop_client_array, img_proc_status, recieve_image_filenames):
     TARGET_MAP_PATH = "static/imgs/"
     AUTONOMOUS_IMAGE_PROCESSING_SAVE_PATH = "static/autonomous_crops"
+    submitted_target_locations = []
     while True:
```

Comment:
Should be a new line before the while loop for spacing and readability.

Reply:
Agreed.